javaagent-instrumentation-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/config/HttpHeadersConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.api.config;
+
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.instrumentation.api.instrumenter.http.CapturedHttpHeaders;
+
+/** Javaagent configuration of captured HTTP request/response headers. */
+public final class HttpHeadersConfig {
+
+  private static final CapturedHttpHeaders CLIENT;
+  private static final CapturedHttpHeaders SERVER;
+
+  static {
+    Config config = Config.get();
+    CLIENT =
+        CapturedHttpHeaders.create(
+            config.getList(
+                "otel.instrumentation.common.experimental.capture-http-headers.client.request"),
```

Comment:
Shouldn't this part of `ExperimentalConfig`?

Reply:
Hmm, you're right. Do you mind if I do that in a separate PR, after this one an https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4255 are both merged? I'll just add another item to the TODO list in the PR description (and create a tracking issue for all of them once this PR is merged)